vela/queue.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package vela
+
+import "github.com/go-vela/types/library"
+
+// QueueService handles retrieving queue from
+// the server methods of the Vela API.
+type QueueService service
+
+// GetQueueCreds fetches queue credentials based valid registration token.
+func (qvc *QueueService) GetQueueCreds() (*library.QueueInfo, *Response, error) {
```

Comment:
```suggestion
func (qvc *QueueService) GetInfo() (*library.QueueInfo, *Response, error) {
```

we usually keep the sdk functions inline with the naming of the api endpoint, for consistency